src/features/distribute/types/index.ts

```diff
@@ -9,3 +9,24 @@ export const DistributionSchema = z.object({
 });
 
 export type Distribution = z.infer<typeof DistributionSchema>;
+
+
+export const PayoutSchema = DistributionSchema.extend({
+  sender: EthAddressSchema,
+  amount: z.number(),
+});
+
+export type Payout = z.infer<typeof PayoutSchema>;
+
+export const PayoutsTablePropsSchema = z.object({
+  distributions: z.array(PayoutSchema),
+});
+
+export type PayoutsTableProps = z.infer<typeof PayoutsTablePropsSchema>;
+
+export const MainTablePropsSchema = z.object({
```

Comment:
Why are you using zod schemas for component props? These are mainly used for validation, for example:
- in routers
- in forms

Reply:
I used that just to fit the code structure